extensions/amp-subscriptions/0.1/platform-store.js

```diff
@@ -487,7 +519,7 @@ export class PlatformStore {
    * @return {!./subscription-platform.SubscriptionPlatform}
    */
   selectPlatformForLogin() {
-    return this.selectApplicablePlatform_(
+    return this.selectApplicablePlatformForFactor_(
```

Comment:
"So, I think login should be scored very simply: service that supports viewer gets to run it, otherwise it must be local."  - It's not clear that this change does that.  It will calculate the score for SUPPORTS_VIEWER but unless the publisher has added an explicit weight it will be zeo and the local will win.

Reply:
No, it doesn't multiply by the score - it only takes into the account the value of the `supportsViewer` factor. So it should pretty much do what you say. In some weird case there could be two services that support viewer and then one of them will be picked essentially in random.

Reply:
Ah yes I see that now.